### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by lazarmilojevic**
added tests, deleted old bad tests.

### CODE REVIEW

File: src/test/java/SpringJPA/AdminPageTest.java [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
+public class AdminPageTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    @WithMockUser(username="User1")
+    public void UserAPICallsAppears() throws Exception{
+        this.mockMvc.perform(get("/user/admin")).andDo(print()).andExpect(status().isOk())
+                .andExpect(content().string(containsString("0/1000")));
```
Comment (namansethi) [inline src/test/java/SpringJPA/AdminPageTest.java:29]: Is it possible to break this into separate parts? It is pretty long command that is hard to understand.

File: src/test/java/SpringJPA/AdminPageTest.java [hunk 2]
```diff
@@ -1,35 +1,40 @@
...
+    }
```
Comment (pruthvichivukula) [inline src/test/java/SpringJPA/AdminPageTest.java]: This should be using An admin type user, not a normal one

File: src/test/java/SpringJPA/AdminPageTest.java [hunk 3]
```diff
@@ -1,35 +1,40 @@
...
+
+    @Test
+    @WithMockUser(username="User1")
+    public void UserIDAppears() throws Exception{
+        this.mockMvc.perform(get("/user/admin")).andDo(print()).andExpect(status().isOk())
```
Comment (namansethi) [inline src/test/java/SpringJPA/AdminPageTest.java:35]: Look at above comment.

File: src/test/java/SpringJPA/AdminPageTest.java [hunk 4]
```diff
@@ -1,35 +1,40 @@
...
+                .andExpect(content().string(containsString("User1")));
+    }
```
Comment (pruthvichivukula) [inline src/test/java/SpringJPA/AdminPageTest.java]: This should be using An admin type user, not a normal one

**Review by pruthvichivukula — CHANGES_REQUESTED** [review]

File: src/test/java/SpringJPA/AdminPageTest.java (hunk 1 above)
Comment (lazarmilojevic) [reply src/test/java/SpringJPA/AdminPageTest.java:29]: This is just how it has to be. Splitting it up would take up so many lines and make the test look like way more than it actually is.

File: src/test/java/SpringJPA/AdminPageTest.java (hunk 3 above)
Comment (lazarmilojevic) [reply src/test/java/SpringJPA/AdminPageTest.java:35]: Look at above resolvation

File: src/test/java/SpringJPA/AdminPageTest.java (hunk 2 above)
Comment (lazarmilojevic) [reply src/test/java/SpringJPA/AdminPageTest.java]: Sorry, I wasn't aware that there were different user types when I pulled from main. I figured I wouldn't need to change that until the role functionality was finalized. I'll make the change.

File: src/test/java/SpringJPA/AdminPageTest.java (hunk 4 above)
Comment (lazarmilojevic) [reply src/test/java/SpringJPA/AdminPageTest.java]: Same as other reply. Got it now. Pushed.

**Review by pruthvichivukula — APPROVED** [review]
test issue was resolved

**Review by namansethi — APPROVED** [review]

**Pull request merged**